test/utils.js

```diff
@@ -66,6 +66,7 @@ describe('utils', () => {
             },
             method: 'POST',
             path: '/',
+            route: '/',
```

Comment:
This isn't right. The `route` object of a request should be more than just a string according to https://github.com/hapijs/hapi/blob/master/API.md#route-public-interface